src/api/SFA.DAS.Provider.Events.Api/Controllers/PaymentsController.cs

```diff
@@ -70,6 +71,31 @@ public PaymentsController(IMediator mediator, ILogger logger)
             }
         }
 
+        [HttpGet]
+        [Route("api/v2/payments/statistics", Name = "PaymentsStatistics")]
+        public async Task<IHttpActionResult> GetPaymentStatistics()
+        {
+            try
+            {
+                var paymentsResponse = await _mediator
+                    .SendAsync(new GetPaymentsStatisticsRequest()
+                   ).ConfigureAwait(false);
+
+                if (!paymentsResponse.IsValid)
+                {
+                    throw paymentsResponse.Exception;

```

Comment:
As mentioned in a previous comment, we shouldn't be handling exceptions in this way.

Reply:
done